libtextsecure/sendmessage.js

```diff
@@ -463,9 +463,10 @@ MessageSender.prototype = {
     const convos = window.getConversations().models;
 
     const conversations = Array.isArray(convos) ? convos : [convos];
+    const currentlyBlockedNumbers = window.BlockedNumberController.getBlockedNumbers();
 
-    const blockedConvos = await libsession.Utils.SyncMessageUtils.filterBlockedNumbers(
-      conversations
+    const blockedConvos = conversations.filter(
```

Comment:
This would fail if say we blocked a number but didn't have a conversation for it. We shouldn't rely on the conversation being there.

Reply:
I see what you mean. It wouldn't crash but we might trigger a sync with a sublist of the actual  ids blocked. I ammended